docs/Extensibility_UG/Intro.md

```diff
@@ -7,12 +7,12 @@
    :hidden:
 
    openvino_docs_Extensibility_UG_add_openvino_ops
-   openvino_docs_Extensibility_UG_GPU
+   openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer
 
 @endsphinxdirective
 
 The Intel® Distribution of OpenVINO™ toolkit supports neural network models trained with multiple frameworks including
-TensorFlow, Caffe, MXNet, Kaldi, PaddlePaddle, and ONNX. The list of supported operations (layers) is different for
+TensorFlow*, PyTorch*, ONNX*, PaddlePaddle*, MXNet*, Caffe, and Kaldi. The list of supported operations (layers) is different for
```

Comment:
Since 2021.4 we are allowed to skip the asterisks in third-party trademarks in our documentation :) We still need to put a proper disclaimer in our template, but we can slowly keep getting rid of them ;)

Reply:
Done